docs/user_guide/execution_model.rst

```diff
@@ -1,75 +1,115 @@
+An example of this might be two parallel synthesis runs with different settings after an import task. The two synthesis "tasks" might be called ``syn0`` and ``syn1``, where:
 
-.. literalinclude:: examples/flowgraph_doe.py
+.. image:: _images/flowgraph_step_index_diagram.png
+   :scale: 40%
+   :align: center
 
-.. image:: _images/flowgraph_doe.svg
+See :ref:`using index for optimization` for more information on why using indices to buid your flowgraph are helpful.
+
+Execution
+^^^^^^^^^^^^^^^
 
-The SiliconCompiler directory structure follows the convention shown below. Each step/index combination ("task") has a separate work directory, with all data inputs read from a local 'inputs' directory and all data outputs placed in a local 'outputs' directory. Reports are placed in the task's local  'reports' directory. Control communication between tasks is done entirely through file based communication. The compilation schema containing all setup parameters and run time metrics are read from 'inputs' directory at the start of a task and written to the 'outputs' directory at the end of a task.
+Flowgraph execution is done through the :meth:`.run()` function which checks the flowgraph for correctness and then executes all tasks in the flowgraph from start to finish.
 
-::
+Flowgraph Examples
+---------------------
 
-    heartbeat
-    └── <jobname>
-        ├── floorplan
-        │   └── <index>
-        │       ├── inputs
-        │       ├── outputs
-        │       └── reports
-        ├── import
-        │   └── <index>
-        │       ├── inputs
-        │       ├── outputs
-        │       └── reports
-        └── syn
-            └── <index>
-                ├── inputs
-                ├── outputs
-                └── reports
+The flowgraph, used in the :ref:`asic demo`, is a built-in compilation flow, called :ref:`asicflow <asicflow-ref>`. This compilation flow is a pre-defined flowgraph customized for an ASIC build flow, and is called through the :meth:`.load_target()` function, which calls a :ref:`pre-defined PDK module <pdk_directory>` that `uses the asicflow flowgraph <https://github.com/siliconcompiler/siliconcompiler/blob/main/siliconcompiler/targets/skywater130_demo.py>`_.
 
+.. rst-class:: page-break
 
-The combination of the flowgraph execution mode and file based communication enables support for distributed processing of any static execution pattern, including serial pipelines, fork-join patterns, and parallel pipelines.
+You can design your own chip compilation build flows by easily creating custom flowgraphs through:
 
-The ubiquitous fork-join execution pattern can be created by adding steps associated with built in function steps to the flowgraph. The built in :ref:`asicflow <asicflow-ref>` is an example of a flow that supports fork-join optimization at every step.
+* :meth:`.node()`/:meth:`.edge()` methods (**recommended**), or
+* :meth:`.set()`/:meth:`.get()` methods
```

Comment:
I think we should not reference this since it's unlikely that it will be done correctly if used.